Initial release candidate (#26)
EquitableLettersSP/ELSP: CODE_OF_CONDUCT.md

Contributor Covenant Code of Conduct written for the development and use for
Open Source Projects, at
[Version 2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).

> This is version 1.0 of our Code of Conduct was adopted on 8 Feb 2021.

Reviewer: ```suggestion
This is version 1.0 of our Code of Conduct, which was adopted on 8 Feb 2021.
```

> This is version 1.0 of our Code of Conduct (ADD DATE AFTER ADDING LINKS).

Reviewer 2: ```suggestion
This is version 1.0 of our Code of Conduct was adopted on 8 Feb 2021.
```

## Our Pledge

> We as editors, reviewers, and writers pledge to follow this code of conduct in the name of our

Reviewer 2: ```suggestion
We as directors, reviewers, and writers pledge to follow this code of conduct in the name of our
```

- Revealing information reviewed in confidence.
- Publishing others' private information, such as a physical or email address,
  without their explicit permission.
- Violation of the confidentiality of the review process without the consent of
  all of the involved parties.

> - Not adhering to the guidelines for submitters and reviews (LINK HERE).

Reviewer 2: Need to add links to those documents here, now that we have them.

Reviewer 2: Postponing this until after the webpage is live.

Reviewer 2: Waiting until after the webpage goes live to put a front-facing link here.
